lsp-mode.el

```diff
@@ -8459,8 +8463,11 @@ archive (e.g. when the archive has multiple files)"
 
 (defcustom lsp-unzip-script (lambda ()
                               (cond ((and (eq system-type 'windows-nt)
-                                          (executable-find "powershell"))
+                                          (executable-find "pwsh"))
```

Comment:
do you have installed this?

https://github.com/PowerShell/PowerShell

pwsh is this!

Reply:
this is powershell7, 
Its executable program name is `pwsh`, while the executable program name of powershell 5 is `powershell`.

Reply:
just like this:

![image](https://github.com/user-attachments/assets/52fe4c14-696c-428a-affa-d23b35d82f49)

Reply:
pwsh is more efficient than powershell

Reply:
Ah, got it. I think we should support both `pwsh` and `powershell` since I assumed most users like me doesn't have `pswh` installed.

Reply:
The next match is to judge powershell
Trust me, most of the windows users are using powershell 7
Because now powershell5 will recommend you to use 7 when you open it.

Reply:
Thanks for the information! I recently re-installed Windows 11, and it still uses powershell and not pwsh. 🤔 But it's nice to know what happens in Powershell's development.

Reply:
https://learn.microsoft.com/en-us/powershell/scripting/whats-new/differences-from-windows-powershell?view=powershell-7.4

this is difference between powershell5 and powershell 7

powershell 5 is just for windows
powershell 7 (pwsh) is for all platforms

Reply:
And pwsh(PowerShell7) is not integrated on windows by default, it needs to be installed manually (download from github or install from ms store)
Starting from Windows 10, PowerShell 5 will be integrated by default

Reply:
Got it! Thank you for your information! :D